src/moduler/aktivitet/visning/avtalt-container/SattTilAvtaltInfotekst.tsx

```diff
@@ -7,10 +7,11 @@ import { useKanSendeVarsel } from './avtaltHooks';
 const getText = (
     kanSendeVarsel: boolean,
     avtaltMedNavMindreEnnSyvDager: boolean,
+    manglerTilDato: boolean,
     forhaandsorienteringstype: ForhaandsorienteringType
 ): string => {
-    if (!kanSendeVarsel) {
-        return 'Aktiviteten er merket "Avtalt med NAV". Forhåndsorientering om konsekvens for ytelse er ikke lagt til aktiviteten.';
+    if (!kanSendeVarsel || manglerTilDato) {
+        return 'Aktiviteten er merket "Avtalt med NAV". Forhåndsorientering er ikke lagt til. Du skal ha informert bruker om mulige konsekvenser for ytelse og dokumentert dette i et samtalereferat.';
```

Comment:
Dukker denne opp hvis man fjerner tildato på en avtalt aktivitet?

Reply:
> Dukker denne opp hvis man fjerner tildato på en avtalt aktivitet?

Nei, SattTilAvtaltVisning tror jeg kun vises som 'kvitterings-side' for forhåndsorientering, så den vil ikke vises ved andre endringer på aktiviteten senere.